solidity/contracts/ECDSARewards.sol

```diff
@@ -182,6 +193,20 @@ contract ECDSARewards is Rewards {
         token.safeTransfer(beneficiary, withdrawableRewards);
     }
 
+    function sendHeartbeat(bytes32 _keep) public {
```

Comment:
What's the gas cost of this function? Assuming it is called for every keep, we should *probably* ensure the minimum reward >> heartbeat cost.

Reply:
For the first call, it should be a few thousand in the necessary queries + 20,000 for the storage. For later updates the overhead goes down by 15,000 because we aren't initializing a previously empty storage slot.